docs/tutorials/gluon/gluon_from_experiment_to_deploymen.md

```diff
@@ -0,0 +1,400 @@
+
+We will use pre-trained ResNet50_v2 model, all you need to do is re-define the last softmax layer for your case. Specify the number of classes in your data and initialize the weights.
+You can also add layers to the network according to your needs.
+
+Before we go to training, one important part is to hybridize your model, it will convert your imperative code to mxnet symbolic graph. It's much more efficient to train a symbolic model,
+and you can also serialize and save the network archietecure and parameters for inference.
+
+```python
+model_name = 'ResNet50_v2'
+finetune_net = get_model(model_name, pretrained=True)
+with finetune_net.name_scope():
+    finetune_net.output = nn.Dense(classes)
+finetune_net.output.initialize(init.Xavier(), ctx = ctx)
+finetune_net.collect_params().reset_ctx(ctx)
+finetune_net.hybridize()
+
+trainer = gluon.Trainer(finetune_net.collect_params(), 'sgd', {
+                        'learning_rate': lr, 'momentum': momentum, 'wd': wd})
+metric = mx.metric.Accuracy()
+L = gluon.loss.SoftmaxCrossEntropyLoss()
+```
+
+### Fine-tuning model on your custom dataset
+
+Now let's define the test metrics and start fine-tuning.
+
+```python
+def test(net, val_data, ctx):
+    metric = mx.metric.Accuracy()
+    for i, batch in enumerate(val_data):
+        data = gluon.utils.split_and_load(batch[0], ctx_list=ctx, batch_axis=0, even_split=False)
+        label = gluon.utils.split_and_load(batch[1], ctx_list=ctx, batch_axis=0, even_split=False)
+        outputs = [net(X) for X in data]
+        metric.update(label, outputs)
+
+    return metric.get()
+
+
+lr_counter = 0
+num_batch = len(train_data)
+
+for epoch in range(epochs):
+    if epoch == lr_steps[lr_counter]:
+        trainer.set_learning_rate(trainer.learning_rate*lr_factor)
+        lr_counter += 1
+
+    tic = time.time()
+    train_loss = 0
+    metric.reset()
+
+    for i, batch in enumerate(train_data):
+        data = gluon.utils.split_and_load(batch[0], ctx_list=ctx, batch_axis=0, even_split=False)
+        label = gluon.utils.split_and_load(batch[1], ctx_list=ctx, batch_axis=0, even_split=False)
+        with ag.record():
+            outputs = [finetune_net(X) for X in data]
+            loss = [L(yhat, y) for yhat, y in zip(outputs, label)]
+        for l in loss:
+            l.backward()
+
+        trainer.step(batch_size)
+        train_loss += sum([l.mean().asscalar() for l in loss]) / len(loss)
+
+        metric.update(label, outputs)
+
+    _, train_acc = metric.get()
+    train_loss /= num_batch
+
+    _, val_acc = test(finetune_net, val_data, ctx)
+
+    print('[Epoch %d] Train-acc: %.3f, loss: %.3f | Val-acc: %.3f | time: %.1f' %
+             (epoch, train_acc, train_loss, val_acc, time.time() - tic))
+
+_, test_acc = test(finetune_net, test_data, ctx)
+print('[Finished] Test-acc: %.3f' % (test_acc))
+```
+
+Note we are able to reach a test accuracy of 93% with only 20 epochs in less than 20 minutes, this is really fast because we used the
```

Comment:
Quantify what type of instance(Mac Pro 2016 OS X 10.x.x/EC2 Instance) you ran the test on, please test with num_workers = cpu_count, you'll be surprised.